snapshotter/Makefile

```diff
@@ -79,6 +76,8 @@ logs:
 
 clean:
 	- rm -f demux-snapshotter
+# Leaving legacy binary cleanup so that existing copies can get to a clean
+# state after pulling from main
 	- rm -f http-address-resolver
```

Comment:
I'm fine not careing existing copies. If we keep this line now, when would we remove the line?